WalletWasabi/JsonConverters/Bitcoin/MoneyBtcJsonConverterNg.cs

```diff
@@ -0,0 +1,29 @@
+using NBitcoin;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace WalletWasabi.JsonConverters.Bitcoin;
+
+public class MoneyBtcJsonConverterNg : JsonConverter<Money>
+{
+	/// <inheritdoc />
+	public override Money? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+	{
+		if (reader.TokenType != JsonTokenType.String)
+		{
+			return null;
```

Comment:
Actually:
`FeeRate`: `legacy` throws `InvalidCastException` / `Ng` throws `JsonException`.
`MoneyBtc`: `legacy` returns `null` / `Ng` returns `null`.

Because in `MoneyBtcConverter` we do `string? stringValue = reader.Value as string;` which can give back `null` in case of an invalid value.
But in `FeeRateConverter` we do `var serialized = (long?)reader.Value;` which will fail with `InvalidCastException`. 
I think this is the closest how we can mimic the behaviour of the `legacy` converter.

We **could** change the `legacy` logic to `long? serialized = reader.Value as long?;` and return `null` in case of string in `Ng` logic, but that would change the original behaviour.

Reply:
you are right as well